Modules/GLO/src/DataCompressionQcTask.cxx

```diff
@@ -0,0 +1,142 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+///
+/// \file   DataCompressionQcTask.cxx
+/// \author Thomas Klemenz
+///
+
+// O2 includes
+#include "Framework/ProcessingContext.h"
+#include <Framework/InputRecord.h>
+
+// QC includes
+#include "QualityControl/QcInfoLogger.h"
+#include "GLO/DataCompressionQcTask.h"
+#include "Common/Utils.h"
+
+// root includes
+#include "TH1.h"
```

Comment:
Doesn't `unique_ptr` need the header? So perhaps move this to the header.

Reply:
It actually worked with the forward declaration in the case of `TH1`. For `TCanvas` the include was needed in the header. I moved the TH1 include to the header now anyway.